spec/controllers/http_status_agent_spec.rb

```diff
@@ -103,10 +105,12 @@ def agent.checked_url
       let(:successful_url) { SecureRandom.uuid }
 
       let(:status_code) { 200 }
+      let(:header) { "Server" }
+      let(:header_value) { "WEBrick" }
```

Comment:
Couldn't figure it out. It should, though.

https://github.com/cantino/huginn/pull/1521#issuecomment-224516245